components/aws/sagemaker/tests/integration_tests/component_tests/test_train_component.py

```diff
@@ -80,3 +80,78 @@ def test_trainingjob(
         assert f"dkr.ecr.{region}.amazonaws.com" in training_image
 
     utils.remove_dir(download_dir)
+
+# @pytest.mark.skipif(utils.get_fsx_id() is None, reason="Skipping FSx Test")
+@pytest.mark.parametrize(
+    "test_file_dir",
+    [
+        pytest.param(
+            "resources/config/fsx-mnist-training", marks=pytest.mark.canary_test
+        )
+    ],
+)
+def test_fsx_trainingjob(
```

Comment:
This can be a separate test file since we are testing the fsx feature here and not the training component.

Reply:
separate question, you dont need this method at all, pass the `"resources/config/fsx-mnist-training"` to `test_trainingjob`

it would look -

```
@pytest.mark.parametrize(
    "test_file_dir",
    [
        pytest.param(
            "resources/config/simple-mnist-training", marks=pytest.mark.canary_test
        ),
        pytest.param(
             "resources/config/fsx-mnist-training", marks=pytest.mark.skipno_fsxid
         )
    ],
)
def test_trainingjob(
```

Reply:
please run `black` in the directory to fix linting issues if any